src/idpi/operators/time_operators.py

```diff
@@ -88,6 +88,36 @@ def delta(field: xr.DataArray, dtime: np.timedelta64) -> xr.DataArray:
     return result
 
 
+def tdelta(field: xr.DataArray, dtime: np.timedelta64) -> xr.DataArray:
+    """Compute weighted difference for a given delta in time.
```

Comment:
could we improve here the name of the operator and documentation so that it reflects better what it does? 
Is this computing an average rate ?

Reply:
Added a paragraph in the docstring

Reply:
thinking again how we could name this operator. Isnt this a moving average difference in time, with a sampling rate defined by dtime ? If so we could rename as something like `tdiff_moving_average` ?

Reply:
It relies on the input being a time averaged value from the start of the computation. I think `tdiff_moving_average` is a bit misleading because it suggests that the operator is performing an averaging aggregation. Maybe `resample_average`? We loose however the correspondence with fieldextra.

Reply:
yes, I this is right. I think `resample_average` is a better name. I think is find to loose the mapping with fieldextra, at some point in time the mapping with fieldextra wont be necessary and it is better to have names that describe well the mathematical operation than applying historical reasons why it was named like that

Reply:
done